### PR TITLE
Rename administrator to registrar

### LINKED ISSUE #1360: Rename `administrator` field to `registrar`
`registrar` sounds better. Also, change `centralized asset` to `regulated asset`.

### PR DESCRIPTION
**Pull request by HoOngEe**
It fixes #1360

### CODE REVIEW

**Comment by sgkim126** [comment]
It fixes https://github.com/CodeChain-io/codechain/issues/1360

File: spec/Transaction.md [hunk 1]
```diff
@@ -43,16 +43,16 @@ The output becomes the lock script hash and parameters of the new asset.
 A permissioned asset is an asset that has an approver.
 This kind of asset needs permission to be transferred.
 
-A centralized asset is an asset that has an administrator.
-The administrator can change the asset scheme and transfer the asset arbitrarily.
+A centralized asset is an asset that has an registrar.
```
Comment (sgkim126) [inline spec/Transaction.md]: Please make it 'regulated asset'

File: state/src/item/asset_scheme.rs [hunk 2]
```diff
@@ -95,7 +95,7 @@ impl AssetScheme {
     }
 
     pub fn is_centralized(&self) -> bool {
```
Comment (sgkim126) [inline state/src/item/asset_scheme.rs]: Please rename it to `is_regulated`.

File: spec/Transaction.md (hunk 1 above)
Comment (HoOngEe) [reply spec/Transaction.md]: Fixed

File: state/src/item/asset_scheme.rs (hunk 2 above)
Comment (HoOngEe) [reply state/src/item/asset_scheme.rs]: Fixed

**Pull request merged**